README.md

```diff
@@ -32,8 +32,10 @@ jobs:
   buf:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
-      - uses: bufbuild/buf-action@v1
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Run Buf
+        uses: bufbuild/buf-action@v1
```

Comment:
Not sure if we want to name are steps for better visibility.

Reply:
Can you test this in the demo repo and show a screenshot of what changes in practice

Reply:
https://github.com/bufbuild/buf-action/actions/runs/10221025753/job/28282633161?pr=62 shows in this CI. can demo

Reply:
Demo here: https://github.com/bufbuild/buf-action-demo/pull/15